common/djangoapps/student/models.py

```diff
@@ -717,6 +717,27 @@ def get_or_create_enrollment(cls, user, course_key):
         return enrollment
 
     @classmethod
+    def get_enrollment(cls, user, course_key):
+        """Returns a CoursewareEnrollment object.
+
+        Args:
+            User : is a Django User object.
+            course_id : is our usual course_id string (e.g. "edX/Test101/2013_Fall)
```

Comment:
A few nitpicks here:

1) Please include the variable name, then the type in parentheses.  For example:
```
user (User): The user associated with the enrollment.
course_id (CourseKey): The key of the course associated with the enrollment.
```

2) Course keys don't always have the form "x/y/z".  We call these "opaque" keys because the format can change over time (for example, we changed it recently to support course reruns).